content/docs/for-buildpack-authors/concepts/package.md

```diff
@@ -3,8 +3,10 @@ title="What is a buildpack package?"
 weight=99
 +++
 
+A `buildpack package` refers to the process of packaging buildpacks for distribution as OCI images or `tar` files.
```

Comment:
```suggestion
A `buildpack package` refers to the process of packaging buildpacks for distribution as OCI images or OCI-compatible `tar` files.
```
You can `docker load` a buildpack package tar, for example

Reply:
updated